src/app/component/eco-news/components/create-news/create-news.component.ts

```diff
@@ -1,9 +1,9 @@
 import { Component, OnInit } from '@angular/core';
 import { Router } from '@angular/router';
-import { FormBuilder, Validators, FormControl, FormGroup, FormArray } from '@angular/forms';
-import { CreateEcoNewsService } from '../../services/create-eco-news.service';
-import { FilterModel, LanguageModel, NewsResponseDTO } from '../../models/create-news-interface';
+import { FormBuilder, Validators, FormControl, FormGroup } from '@angular/forms';
 import { MatDialog } from '@angular/material/dialog';
+import { CreateEcoNewsService } from '@eco-news-service/create-eco-news.service';
+import { FilterModel, NewsResponseDTO } from '@eco-news-models/create-news-interface';
 import { CreateNewsCancelComponent } from '../../../shared/components/create-news-cancel/create-news-cancel.component';
```

Comment:
long-doted path - improve it

Reply:
i cant understand.
the path of CreateNewsCancelComponent is: "src/app/component/shared/components/create-news-cancel/create-news-cancel.component".
in tsconfig.json i create new alias: "@shared/*": ["src/app/component/shared/*"],
and instead of long-dotted path I write:
import { CreateNewsCancelComponent } from '@shared/components/create-news-cancel/create-news-cancel.component';
and it doesnt work, i have an error:
./src/app/component/eco-news/components/create-news/create-news.component.ts
Module not found: Error: Can't resolve '@shared/components/create-news-cancel/create-news-cancel.component' in 'C:\Users\Lnv\Desktop\Angular\GreenCityClient\src\app\component\eco-news\components\create-news'

Reply:
You aren't first who created aliases). Look how other did it, figure it out and fix this issue.

Reply:
finally fixed.... all i had to do is just restart project :(

Reply:
finally fixed... al i had to do is to restart the project